modules/core/src/main/java/org/apache/ignite/internal/managers/deployment/GridDeploymentPerVersionStore.java

```diff
@@ -376,7 +396,10 @@ else if (ctx.discovery().node(meta.senderNodeId()) == null) {
                     return null;
                 }
 
-                dep = (SharedDeployment)searchDeploymentCache(meta);
+                if (!skipSearchDeployment)
+                    dep = (SharedDeployment)searchDeploymentCache(meta);
+                else
+                    dep = null;
```

Comment:
We can assignt the variable as null at definition.

Reply:
done thanks